src/test/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResourceTest.java

```diff
@@ -103,6 +107,7 @@ void setup() {
     FormRPartAResource formRPartAResource = new FormRPartAResource(service, validator);
     mockMvc = MockMvcBuilders.standaloneSetup(formRPartAResource)
         .setMessageConverters(jacksonMessageConverter)
+        .addInterceptors(new TraineeIdentityInterceptor(new TraineeIdentity()))
```

Comment:
This feels quite hacky, but it was the only way I could get a the interceptor preHandle() to work.

Reply:
I've pushed an additional commit with some refactoring for this and the Part B equivalent.